WTWSMS/localisation/00_WtWSMS_override.csv

```diff
@@ -212,5 +212,61 @@ d_scottish_band_adj;Pictish;
+EVTOPTA67010;Demand allegiance to the Throne of Saint Peter;;;;;;;;;;;;;x
+EVTTOOLTIP67011;will listen to the demands of the [From.GetTitle];;;;;;;;;;;;;x
+EVTTOOLTIP67012;Our mortal liege will hear of this.;;;;;;;;;;;;;x
+EVTOPTB67010;Let it slide;Laisser faire;Die Sache sein lassen;;Ignor�moslo;;;;;;;;;x
+EVTDESC67011;An emissary is visiting us from the [From.GetTitle], giving us gifts and demanding our allegiance to the holy Church.;;;;;;;;;;;;;x
+EVTOPTA67011;[Root.Religion.GetHighGodName] is great!;;;;;;;;;;;;;x
+EVTOPTB67011;My liege is my worldly sovereign;M[Root.Liege.GetOnA] [Root.Liege.GetLordLady] lige est m[Root.Liege.GetOnA] souverain[Root.Liege.GetNoneE].;Mein Souver�n ist mein Lehnsherr;;Mi se�or es mi soberano;;;;;;;;;x
+EVTDESC67012;[From.GetTitledFirstName] has been approached by their spiritual head, and is clearly disregarding our lawful rights.;;;;;;;;;;;;;x
+EVTOPTA67012;Will no one rid me of this turbulent [Root.Religion.GetPriestTitle]?;;;;;;;;;;;;;x
+EVTDESC67020;The Kings and Emperors refuse to follow the Law of God. However, any nobleman anointed by God's servants can become a rightful King.;;;;;;;;;;;;;x
+EVTOPTB67020;That is a bad idea...;C'est une mauvaise id�e...;Das ist keine gute Idee ...;;Es una mala idea...;;;;;;;;;x
+EVTTOOLTIP67021;will evaluate our proposal;va �valuer notre proposition.;wird unseren Vorschlag abw�gen;;Estudiar� nuestra propuesta;;;;;;;;;x
+EVTDESC67021;The [From.GetTitle] is asking us to replace our liege, and is giving us the divine permission to do so.;;;;;;;;;;;;;x
+EVTOPTA67021;God save the [From.GetTitle]!;Que Dieu sauve le Pape�!;Gott besch�tze den Papst!;;�Dios salve al Papa!;;;;;;;;;x
+EVTOPTB67021;I am loyal to my liege!;Je suis fid�le � m[Root.Liege.GetOnA] [Root.Liege.GetLordLady] lige�!;Ich bleibe meinem Lehnsherrn treu.;;�Soy leal a mi se�or!;;;;;;;;;x
+EVTTOOLTIP67022;Our Liege will hear of this.;M[Root.Liege.GetOnA] [Root.Liege.GetLordLady] lige va en �tre inform�[Root.Liege.GetNoneE].;Unser Lehnsherr wird hiervon erfahren.;;Nuestro se�or se enterar� de esto.;;;;;;;;;x
+EVTDESC67022;[From.GetTitledFirstName] is a rebel and a traitor. The false sweet words of the false pope has swayed [From.GetHerHim].;[From.GetTitledFirstName] est coupable de r�bellion et de tra�trise. Les fausses belles paroles du faux Pape l'ont influenc�[From.GetNoneE].;[From.GetTitledFirstName] ist rebellisch und verr�terisch! Die s��en Heucheleien des falschen Papstes haben [From.GetHerHim] eingelullt.;;[From.GetTitledFirstName] es un rebelde y un traidor. Le han influido las falsas y dulces palabras del falso Papa.;;;;;;;;;x
+EVTOPTA67022;[From.GetSheHeCap] will pay for this!;[From.GetIlElleCap] va le payer�!;[From.GetSheHeCap] wird daf�r bezahlen!;;�[From.GetSheHeCap] pagar� por esto!;;;;;;;;;x
+SUCH_VERY_WOW;Serene Doge;;;;;;;;;;;;;x
+MUCH_WOW;Doge;;;;;;;;;;;;;x
+DOGE_WOW;Serene Doge;;;;;;;;;;;;;x
```

Comment:
![image](https://user-images.githubusercontent.com/29546927/28239324-6f54b13c-696a-11e7-8913-e74fd54ab39b.png)
Is it an easter-egg?